main_module/static/css_files/user-information-form.css

```diff
@@ -0,0 +1,53 @@
+body{
+    background-color: #181818;

```

Comment:
kolory warto trzymac w zmiennych
a tak ogolnei to cssa nie oceniam bo sie nei znam